text/0113-secure-script-container.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Secure Script Container
+status: DRAFTED
+created_at: 2020-07-15
+pr: https://github.com/salesforce/lwc-rfcs/pull/38
+champion: Caridy Patiño, Ted Conn
+---
+
+# Secure Script Container for LWC Apps
```

Comment:
They are secure because they are running inside a secure javascript sandbox. I'm not tied to the name but `<lwc-script>` implies this is an LWC-only concept and it actually would work for any web component application.

Reply:
yeah, this is misleading for sure. We will change the name from `secure` to `virtual` since the goal here is not to secure anything, but to virtualize it.

Reply:
ok, changes are in, please, take another look.